frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/commentsPanel/comment/currentUserComment/currentUserComment.component.tsx

```diff
@@ -67,10 +66,10 @@ export const CurrentUserComment = ({
 	return (
 		<CommentWithButtonsContainer>
 			<CommentButtons>
-				<ErrorCommentButton onClick={() => onDelete(_id)}>
+				<ErrorCommentButton onClick={onDelete}>
 					<DeleteIcon />
 				</ErrorCommentButton>
-				<PrimaryCommentButton onClick={() => onReply(_id)}>
+				<PrimaryCommentButton onClick={onReply}>
```

Comment:
👀 (if it complicates the code dont do it)
```suggestion
				<PrimaryCommentButton onClick={onReply(comment...)}>
```